account_wallet/security/security.xml

```diff
@@ -53,4 +53,15 @@
         <field name="perm_unlink" eval="0" />
     </record>
 
+    <!-- Wizard -->
+    <record model="ir.model.access" id="account_move_refund_wallet">
+        <field name="name">wizard.account_move_refund.wallet user</field>
+        <field name="model_id" ref="model_wizard_account_move_refund_wallet" />
+        <field name="group_id" ref="account.group_account_invoice" />
```

Comment:
Don't we have a group for wallet operations ?

Reply:
i can use access_account_wallet_manager ?<br/>
@rousseldenis  ?

Reply:
This is a too big change here.

That can be done in another PR